docker/tests/sidecar/mssql-docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# https://docs.docker.com/compose/compose-file/compose-versioning/
```

Comment:
Let's try to avoid duplication of these db-type docker compose files and move them to a common folder for integration and postman test and re-use them. This applies for each `{db-type}-docker-compose` and `open-distro-docker-compose`